app/models/person.rb

```diff
@@ -8,4 +8,17 @@ class Person < ApplicationRecord
   def full_name
     [first_name, last_name].compact.join(' ')
   end
+
+  def full_address
+    return address unless split_address?
+    address_data.values.reject(&:empty?).join(', ')
```

Comment:
I would change this to blank?, because that protects us from possible `nil` values in the hash:

```ruby
> ['', nil, ' '].reject(&:blank?)
=> []
> ['', nil, ' '].reject(&:empty?)
NoMethodError: undefined method `empty?' for nil:NilClass
from (pry):5:in `reject'
```